docs/examples/003-area-calc.ipynb

```diff
@@ -0,0 +1,360 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "# Face Area Calculations\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "This notebook will showcase the different area calculation options provided by UXarray\n",
+    "the grid topology attributes and variables stored in the `UXarray.Grid`\n",
```

Comment:
"... the grid topology ..." may need to be rephrased, was the intent to say something like ""... through grid topology ...""?